custom_typings/merge-stream.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module "merge-stream" {
```

Comment:
I actually just looked at this again and we don't actually need a `merge-stream` declaration (only used in gulpfile.js). Deleting this file but created https://github.com/DefinitelyTyped/DefinitelyTyped/pull/11452 to fix upstream